scripts/views/eventView.js

```diff
@@ -9,12 +9,41 @@
   eventHandler.lookup = function (e) {
     e.preventDefault();
     var zip = $('#look-up input').val();
+    if (zip && zip.length === 5) {
```

Comment:
Mind if I suggest a different approach here?  

Instead of using regex during the parsing let's limit the input itself with `<input pattern="regexp">` and limit it to 10 char d and -?  That way we don't have to worry about if parsing it afterward AND it'll stop users from entering letters etc 

Then we can just use `$('#look-up input').val().split(',')` and boom we're done.  Save some code.